NewBank/newbank/server/Account.java

```diff
@@ -1,14 +1,28 @@
 package newbank.server;
 
+import java.time.LocalDate;
+
 public class Account {
 
+    private int accountNumber;
```

Comment:
Do we need this field if we agreed before that the account will have a unique name?

Reply:
I think this might refer to the account number ticket that James was working on? Each account will have a number assigned to it (I think based on how many bank accounts are in the system). This makes it more similar to actual banks, where on each bank card we have an account number

Reply:
That makes sense Amy, thanks.